include/string.h

```diff
@@ -50,15 +50,21 @@ void *memchr (const void *src : itype(_Array_ptr<const void>) byte_count(n),
               int c,
               size_t n)
   : itype(_Array_ptr<void>) byte_count(n);
-
 char *strcpy (char *__restrict, const char *__restrict);
-char *strncpy (char *__restrict, const char *__restrict, size_t);
+// Copies the first n characters of source to destination. If the end of the source C string (which is signaled by a null-character) is found before num characters have been copied, destination is padded with zeros until a total of num characters have been written to it.
+char *strncpy(char *__restrict d : itype(__restrict _Nt_array_ptr<char>) count(n),
```

Comment:
Refer http://www.cplusplus.com/reference/cstring/strncpy

The src should not have any bounds because by definition this function copies n  characters from src to dest and it does not matter what the size of src is.

Also I think the dest should be an array_ptr not be an nt_array_ptr because:
`No null-character is implicitly appended at the end of destination if source is longer than num. Thus, in this case, destination shall not be considered a null terminated C string (reading it as such would overflow).
`